src/decorator-annotator.ts

```diff
@@ -279,7 +280,7 @@ class DecoratorRewriter extends Rewriter {
     super(sourceFile);
   }
 
-  process(): {output: string, diagnostics: ts.Diagnostic[]} {
+  process(): {output: string, diagnostics: ts.Diagnostic[], sourceMap: SourceMapGenerator} {
```

Comment:
Why are these returning Generators rather than the SourceMaps themselves?  (Am I just misunderstanding the API?)